lib/RequireJsStuffPlugin.js

```diff
@@ -18,27 +19,11 @@ module.exports = class RequireJsStuffPlugin {
 				if(typeof parserOptions.requireJs !== "undefined" && !parserOptions.requireJs)
 					return;
 
-				function remove(expr) {
-					var dep = new ConstDependency(";", expr.range);
-					dep.loc = expr.loc;
-					this.state.current.addDependency(dep);
-					return true;
-				}
-				parser.plugin("call require.config", remove);
-				parser.plugin("call requirejs.config", remove);
+				parser.plugin("call require.config", ParserHelpers.toConstantDependency(";"));
+				parser.plugin("call requirejs.config", ParserHelpers.toConstantDependency(";"));
```

Comment:
Good point. But we could do this in a seprate PR, as this also require tests...

Do you want to open one?